sed/calibrator/energy.py

```diff
@@ -1442,8 +1442,10 @@ def align_dld_sectors(
             dask.dataframe.DataFrame: Dataframe with the new columns.
             dict: Metadata dictionary.
         """
-        sector_delays = sector_delays or self.sector_delays
-        sector_id_column = sector_id_column or self.sector_id_column
+        if sector_delays is None:
+            sector_delays = self.sector_delays
+        if sector_id_column is None:
+            sector_id_column = self.sector_id_column
```

Comment:
@steinnymir This construction produced an error for me, once you do not actually pass the same array (reference). Then the truth value of this comparison is ambigous. So I suggest to generally stick with comparison to None.